pelita/game.py

```diff
@@ -669,28 +675,30 @@ def apply_move(gamestate, bot_position):
             food[1 - team].remove(bot_position)
             # This is modifying the old game state
             score[team] = score[team] + 1
-    # check if anyone was eaten
+    # check if we killed someone
     if bot_in_homezone:
         killed_enemies = [idx for idx in enemy_idx if bot_position == bots[idx]]
         for enemy_idx in killed_enemies:
             _logger.info(f"Bot {turn} eats enemy bot {enemy_idx} at {bot_position}.")
             score[team] = score[team] + KILL_POINTS
             init_positions = initial_positions(walls)
             bots[enemy_idx] = init_positions[enemy_idx]
-            gamestate['respawned'][enemy_idx] = True
-            deaths[abs(team-1)] = deaths[abs(team-1)] + 1
-            _logger.info(f"Bot {enemy_idx} respawns at {bots[enemy_idx]}.")
+            kills[turn] += 1
+            deaths[enemy_idx] += 1
+            bot_eaten[enemy_idx] = True
+            _logger.info(f"Bot {enemy_idx} reappears at {bots[enemy_idx]}.")
     else:
-        # check if bot was eaten itself
+        # check if we have been eaten
         enemies_on_target = [idx for idx in enemy_idx if bots[idx] == bot_position]
         if len(enemies_on_target) > 0:
             _logger.info(f"Bot {turn} was eaten by bots {enemies_on_target} at {bot_position}.")
             score[1 - team] = score[1 - team] + KILL_POINTS
             init_positions = initial_positions(walls)
             bots[turn] = init_positions[turn]
-            gamestate['respawned'][turn] = True
-            deaths[team] = deaths[team] + 1
-            _logger.info(f"Bot {turn} respawns at {bots[turn]}.")
+            deaths[turn] += 1
+            kills[enemies_on_target[0]] += 1
```

Comment:
We should award every bot half a point here ;)

Reply:
Seriously, we discussed about it and we decided that this is the less painful solution. (Having floating-point points would explode the universe, I think)